src/ess/reduce/time_of_flight/simulation.py

```diff
@@ -10,6 +10,8 @@
 
 def simulate_beamline(
     choppers: Mapping[str, DiskChopper],
+    source_position: sc.Variable,
+    *,
```

Comment:
```suggestion
    *,
    choppers: Mapping[str, DiskChopper],
    source_position: sc.Variable,
```

use kwargs only?

Reply:
The first arg has a distinct type, but sure.